src/commands/profiles/add.ts

```diff
@@ -0,0 +1,69 @@
+import { runCommand } from '../../util/command';
+import { outputFormat } from './util';
+const open = require('open')
+
+const jobTemplateKinds = [
+	'folder_mapping',
+	'personal_drive',
+	'transfer',
+]
+
+function isJobTemplateKindValid(jobTemplateKind) {
+	if (jobTemplateKinds.indexOf(jobTemplateKind) > -1) {
+		return true
+	}
+	return false;
+}
+
+function getJobTemplatesJson(argv, output) {
+	return argv.templates && argv.templates.split(',').map(input => {
+		const jobTemplateInputParsed = input.trim().split(':');
```

Comment:
Default to 'transfer' type, which will allow you to use destructuring...

`const [jobTemplateId, jobTemplateKind = 'transfer'] = input.trim().split(':');`

Reply:
will do!